src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java

```diff
@@ -31,4 +31,14 @@ public static boolean isDateWithinLast3Months(LocalDate compareToDate, String qu
         ApiLogger.infoContext(loggingContext, qualifiedFieldName + " must be in the past 3 months");
         return false;
     }
+
+    public static boolean isCeasedDateAfterStartDate(LocalDate ceasedDate, LocalDate startDate, String qualifiedFieldName, Errors errors, String loggingContext) {
```

Comment:
what if they are equal? Not before does not always mean after. Not sure they can cease on appointed date - probably someone trying to "remove" a BO so important to get that right.

Reply:
Good point. Method now checks that ceased date is neither before nor the same as start date